packages/ui-logic/src/balances.ts

```diff
@@ -40,3 +40,12 @@ export function amountToGwei(amount: Amount): string {
   const display = Number.isInteger(value) ? value : value.toFixed(2);
   return `${display} Gwei`;
 }
+
+export function sumToString(amountA: Amount, amountB: Amount): string {
```

Comment:
I think we should have separate function to sum two `Amounts` and then if we want, convert it to the string we should use `amountToString` function. Please update this function signature to `function sumToString(amountA: Amount, amountB: Amount): Amount` and use it to sum two `Amounts`. Then if you want to convert it to string simply use `amountToString`.

Reply:
Also please change function name, something like `sumAmounts`.